### PR TITLE
Setup basic mailroom

### PR DESCRIPTION
**Pull request by gmittal**
Wrote most of the core functions that will be needed to bridge the connection gap between volunteers and users.

@ankitr, PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,23 @@ HASH_LENGTH=XXXX
 HASH_SALT=XXXXXXXXXXXXXXXX
 RETHINK_HOSTNAME=[0.0.0.0|127.0.0.X|localhost|etc.]
 RETHINK_PORT=XXXXX
+TWILIO_ACCOUNT_SID=XXXXXXXXXXXXXXXX
+TWILIO_AUTH_TOKEN=XXXXXXXXXXXXXXXX
+TWILIO_PHONE_NUMBER=XXXXXXXXXXX
+SENDGRID_API_KEY=XXXXXXXXXXXXXXXXXXXX
+SENDGRID_EMAIL=hello@XXXXX.XXXX
```
Comment (ankitr) [inline README.md:18]: Add these to `configure.sh`.

File: templates/message.html [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
...xub13qWiNJQovb5vyHeTKpzbFYq81NqyG2peNy0/H6LKjKJrXJNe33vUMP1bg6ekdNj6Z3DQ/725BfxvD871dxIUIuR/5tnqXVHVNWk63XG+glEwtlidyqipiZKvP7dKzos41yn5smTMM7v+nPcC/19smXeEP+x7kuuSGqu87TZx9mhp4Jn7OMiXnLyi/JFRANf3OVbecOTDU7f1WD1fcV4oTU9F2XOOytriXWVZY3xqPftzYr2TXK7V2ulPq9qqce7ho/yJWTyZpoXuXPFcgAAeNpt0DdsU3EQx/HvJY6dOL330Hvze7ZT6HaSR++9E0hcICTBwUBoAdGrQEgwgWgLIHoVCBgA0ZsoAgZmuhiAFZy8Pxu3fHQn3en0I4rW+uPDx//qM0iURBONhRis2IglDjvxJJBIEsmkkEoa6WSQSRbZ5JBLHvkUUEgRbWhLO9rTgY50ojNd6Eo3utODnvSiN31woKHjxIWbYkoopYy+9KM/AxjIIAbjwUs5FVRiMIShDGM4IxjJKEYzhrGMYzwTmMgkJjOFqUxjOjOYySxmM4e5VImFo2xgIzfYx0c2sYvtHOA4xySGbbxnPXvFKjZ2sp8t3OaDxHKQE/ziJ785wikecI/TzGM+u6nmETXc5yHPeMwTnvIpkt5LnvOCM/j5wR7e8IrXBPjCN7aygCALWUQtdRyinsU0EKKRMEtYyrJIystZQRMrWc0qrnKYZtawlnV85TvXOMs5rvOWdxIndomXBEmUJEmWFEmVNEmXDMmULM5zgctc4Q4XucRdNnNSsrnJLcmRXHZInuRLgRRKkdVf29QQ0GzhuqDD4agw9TiUqvfqSqeyrEU9sqDUlLrSqXQp3cpiZYmyVPnvnsdUU3c1ze4L+sOhmuqqxoA50g1Tt2GpDIfqWxu3Ud6i4TX/iKgrnUrXXwhgnS8AeNo9zb0OwVAYxvEebU8/9LtlIzU0xEmEGzBply4iEm3iOswWI27lrcnuwnjCcbbn9yz/J3tfiF21muxt0zF2a7uKi2ZCUVtTtsM4tyPi4thopOcl6WJDRl4+9GFPfGECxh8cMF8SFsDvEjZgHSQcwF5LuIBTSPQBd/EDI08Gfbzesic6vTqBAeiPFUMw2CtGYLhSjMForpiA8UwxBZOpYgamheIAzBRbysQHzsRVBgAAAVV+7WIAAA==) format('woff');
```
Comment (ankitr) [inline templates/message.html:70]: Can we use google's hosting instead of loading it ourselves?

File: README.md (hunk 1 above)
Comment (gmittal) [inline README.md:18]: Alright.

File: templates/message.html (hunk 2 above)
Comment (gmittal) [inline templates/message.html:70]: I believe we can? Not sure. There are a lot of issues with email templates allowing fonts (very strict regulations). I'll test it out.

Comment (ankitr) [inline templates/message.html:70]: I didn't notice this was for the email templates. Try it out and let me know, but if not it's fine.

Comment (gmittal) [inline templates/message.html:70]: Importing the external CSS from Google is no different than loading it as base64.

File: README.md (hunk 1 above)
Comment (gmittal) [inline README.md:18]: Made the fix. LGTY @ankitr?

**Comment by ankitr** [comment]
LGTM. Merge away.

**Pull request merged**